### PR TITLE
Correct typos

### PR DESCRIPTION
**Pull request by mrosecrance**
The AZ doc had a few odd grammar phrasings or extra veriage. Removed to hopefully help readability

### CODE REVIEW

File: content/azs.md [hunk 1]
```diff
@@ -100,16 +100,16 @@ VMs total: 4
 
 ---
 
-## Adding and removing AZs do a deployment.
+## Adding and removing AZs to a deployment.
 
 > **_NOTE:_**
->   With some CPIs AZS are referenced in Subnet configs of the cloud-config networks block. This means that moving AZs can have networking implications. E.g. networks can be Zonal ( e.g. AWS Subnets are associated to a Zone) or Regional ( e.g. GCP Subnets are associated with a Region). This is important to keep in mind when planning an AZ migration.
+>   With some CPIs AZS are referenced in the Subnet configuration of the cloud-config networks block. This means that moving AZs can have networking implications. E.g. networks can be Zonal ( e.g. AWS Subnets are associated to a Zone) or Regional ( e.g. GCP Subnets are associated with a Region). This is important to keep in mind when planning an AZ migration.
```
Comment (bgandon) [inline content/azs.md:106]: We can do a bit better, removing extra spaces next to parenthesis.

Plus, the “_referenced in the Subnet configuration of the cloud-config networks block_” can still be improved.

Something like “_With some CPIs, availability zones are tied to subnets, as specified by the networks section of the Cloud Config_”

File: content/azs.md [hunk 2]
```diff
@@ -100,16 +100,16 @@ VMs total: 4
...
 
 
 > **_GENERAL LIMITATIONS:_** 
-> - bosh does not migrate persistent disk contents cross AZ. Persistent disks attached to a vm that is moved to another AZ get orphaned and eventually deleted by bosh.
-> - singleton instances will face a downtime while being recreated. If they have persistent disks attached the data on the disk will not be migrated.
-> - moving a vm with a staticly assigned IP addr will fail. Bosh will create the new instance before deleting the old instance. This means that at creation time of the new instance, the static IP is still attached to the old instance.
+> - bosh does not migrate persistent disk contents across AZs. Persistent disks attached to a vm that is moved to another AZ will be orphaned and eventually deleted by bosh.
```
Comment (bgandon) [inline content/azs.md:110]: At least capitalize “Bosh”

**Review by bgandon — CHANGES_REQUESTED** [review]
That’s already nice and good catches, but we can do a bit better, and avoid having to come back to it later.

**Review by ramonskie — APPROVED** [review]

**Comment by rkoster** [comment]
@mrosecrance can you take a loot at the changes requested by @bgandon

**Comment by rkoster** [comment]
ping @mrosecrance

**Comment by rkoster** [comment]
@bgandon, we are gonna go ahead an merged this PR, because it already improves upon the current state of the docs. If you would like the further improve it even more please create a separate PR.

**Pull request merged**